qemu/tests/blockdev_inc_backup_with_ignore.py

```diff
@@ -26,6 +26,7 @@ def _create_inc_dir(self):
                            ignore_status=False, shell=True)
         except:
             self._clean_inc_dir()
+            self.test.fail("Fail to create test env with lvm")
```

Comment:
what about using 'raise' instead, then we can see the exact error msg from exception

Reply:
Have updated, please help to check again.